--- a/bin/vsnp3_step1.py
+++ b/bin/vsnp3_step1.py
@@ -155,10 +155,10 @@
         alignment.excel(self.excel_stats.excel_dict)

         #FASTQ usability
-        if alignment.zero_coverage.ave_coverage < 40 or alignment.DUPLICATION_RATIO > .80 or float(fastq_stats.R1.passQ20) < 50.0:
+        if alignment.zero_coverage.ave_coverage < 40 or alignment.DUPLICATION_RATIO > .80 or float(str(fastq_stats.R1.passQ20).replace(',', '')) < 50.0:
             fastq_usability = 'Poor'
             color_quality = bcolors.RED
-        elif alignment.zero_coverage.ave_coverage < 70 or alignment.DUPLICATION_RATIO > .10 or float(fastq_stats.R1.passQ30) < 70.0:
+        elif alignment.zero_coverage.ave_coverage < 70 or alignment.DUPLICATION_RATIO > .10 or float(str(fastq_stats.R1.passQ30).replace(',', '')) < 70.0:
             fastq_usability = 'Questionable'
             color_quality = bcolors.YELLOW
         else:
